/**:
  detector:
    ros__parameters:
      timer_period: 0.05
      nms_threshold: 0.35
      prob_threshold: 0.35
      target_size: 480
      weights: "best100ep1inp.ncnn" #best100ep1inp.ncnn best80ep.ncnn
      focal_length: 3.15
      width_side: 95.0
      width_front: 68.0
      K: 1.20
      Ti: 100.0
      Td: 0.07
      linearSpeed: 0.05
      stop_from_planner: 0.7
      stop_rotate: 0.0055
      stop_docking: 0.15

  distances:
    ros__parameters:
      timer_period: 1.0
      nms_threshold: 0.35
      prob_threshold: 0.35
      target_size: 480
      weights: "best100ep1inp.ncnn" #best100ep1inp.ncnn best80ep.ncnn
      focal_length: 3.15
      width_side: 95.0
      width_front: 68.0

  searching_shuttlecock_action_server:
    ros__parameters:
      timer_period: 0.02

  rotate_action_server:
    ros__parameters:
      stop_rotate: 0.0055
      K: 1.20
      Ti: 100.0
      Td: 0.07
      timer_period: 0.2

  pick_shuttlecock_action_server:
    ros__parameters:
      stop_docking: 0.15
      linearSpeed: 0.05
      timer_period: 0.2

  get_to_shuttlecock_action_server:
    ros__parameters:
      stop_from_planner: 0.7
      timer_period: 0.2